### PR TITLE
updated psra 3857 hexgrids

### PR DESCRIPTION
**Pull request by wkhchow**
- includes the revised epsg 3857 hexgrid indicators for psra

Sorry, I did not include it in the previous PR. I was looking at the release 1.0.0 and it included the hexgrids, so I think this is the right time to get that included with the 1.0.1 release? Along with the previous PR (including the total values)

### CODE REVIEW

**Comment by wkhchow** [comment]
Adding a bit more context to this PR for clarity.
- Original assets were created from model-factory/opendrr-api 1.4.1 release (https://github.com/OpenDRR/model-factory/releases/tag/v1.4.1 https://github.com/OpenDRR/boundaries/releases/tag/v1.4.0) which includes hexgrids that were created with epsg 4326 WGS84 coordinates. 
- This new PR has the updated hexgrids that were generated for model-factory/opendrr-api 1.4.4 release which now the hexgrids are generated in epsg 3857 (web mercator, google maps). (https://github.com/OpenDRR/model-factory/releases/tag/v1.4.4 https://github.com/OpenDRR/boundaries/releases/tag/v1.5.0)

**Review by tieganh — APPROVED** [review]

**Comment by tieganh** [comment]
Approved but will leave it to you to merge @wkhchow

**Comment by wkhchow** [comment]
Merged! Will create release asset after Tiegan's PR 92 is merged in as well.

**Pull request merged**
